pkg/test/framework/components/stackdriver/google.go

```diff
@@ -75,7 +76,7 @@ var (
 			resourceType: "k8s_pod",
 		},
 	}
-	queryInterval = -5 * time.Minute
+	queryInterval = -90 * time.Second
```

Comment:
Any reason for this? seems reasonable since we shouldn't need 5min but am curious

Reply:
We recently added some variant to the access log test (audit log). Fetching with too wide time interval + limit number of entries returned would make the returned entries all from previous test cases.

Reply:
Can we add some other distinguisher? Assuming our tests are slow so there won't be overlap is not great...

Reply:
Yeah we should be able to filter by log name, this PR is just to make the least amount of change to make test pass :). I will add that in a follow up PR.

Reply:
@howardjohn the logs query is based on namespace. do we feel that the echo ns will be reused within a time period?

Reply:
In real tests it shouldn't since its randomized. However, for local runs `--istio.test.stableNamespaces` removes randomization and is super useful for quick iteration.

Reply:
Ok. The tricky bit is that logName is not configurable for Stackdriver. Making that configurable, just for testing, is a big lift.